README.md

```diff
@@ -165,10 +165,11 @@ While some implicit technical details are obvious to you, they might not be for
 
 ## Code examples
 
-* try to keep code examples together in one block, avoid mixing paragraphs and code blocks
+* Try to keep code examples together in one block, avoid mixing paragraphs and code blocks
 * Code blocks follow standard markdown formatting
 * Show output only when necessary
 * Output should be a comment
+* Use comments in between commands to explain what's going on
```

Comment:
I do not agree with this. Code comments should say why, not what because the code says what.

For example....
If the reader can not understand that "$ git clone URL" means "#clone remote repo to local folder", then they are not the intended audience for this article. No amount of comments in the code block will help them. If you as the reader don't understand a trivial example like "cd directory", then use google or some other platform to grasp the basics, then come back and try to host a website. Or use a different wysiwyg-website service. A certain level of proficiency must be assumed, writing help files for people with 65 IQ is not worthwhile, neither is assuming that people are completely incapable of googling. 

--- 

This style (below) is absolute garbage. I have formed this opinion over several years and it will take some serious convincing to make me change my mind on this. The fact that numerical lists look ugly (because of crazy negative left margin) in our help is a different matter.

```
# 1. Clone the (empty) repository
git clone {{ssh-user}}@deploy.{{region}}.frbit.com:{{app-name}}.git

# 2. Go to the app-name folder
cd {{app-name}}

# 3. Do stuff
echo '<?php echo "PHPower to the PHPeople";' >index.php

# 4. Initialize Git locally
git add index.php

# 5. Commit changes
git commit -am 'Initial commit'

# 6. Set upstream and 1st push
git push -u origin master

# 7. Every deploy from now on
git push

```
Any half decent textbook about code will list out code samples with line numbers, and if complex ideas need explaining, the author will refer to codelines individually and explain what is going on. Mixing explanations with code is poor form and distracting. Further, commenting trivial examples conveys to the reader that the author has no clue about the subject matter. I prefer reading authors who know more than me... When trivial examples are complemented by _incorrect_ comments, then for me, the whole page can just go and have some pancakes - I will never return to learn anything else on that page: 

    # 4. Initialize Git locally
    git add index.php

git init does the initialization. A cloned repo is already initialized. I am convinced that useless style breeds useless comments. This is a good example of that.

    # 5. Commit changes
    git commit -am 'Initial commit'

Annotating a commit (-a) in a trivial example like this is Stupid, because you are potentially confusing the reader with information that is irrelevant.

    # 6. Set upstream and 1st push
    git push -u origin master
    
Pointless comment. Upstream branch is implied when you clone. This comment bloats the example for no reason and provides potentially confusing information.